src/modules/Dashboard/features/Managers/Managers.tsx

```diff
@@ -222,6 +230,9 @@ export const Managers = () => {
       if (searchValue) {
         return t('noMatchWasFound', { searchValue });
       }
+      if (!canViewTeam) {
+        return t('noPermissions');
+      }
```

Comment:
Should we be using the `noPermissionsComponent` that's returned from the `usePermission` hook,  for consistency? There's actually a specific icon that it uses, and that component is already available as a variable here. I just clocked that when you switch between the Applet Overview tab and the Applet Team tab, the variation in styling of "no permissions" state is noticeable:

| Overview | Team |
| - | - |
| <img width="1713" alt="Screenshot 2024-06-17 at 11 40 35 AM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/4c6c88ba-5511-4054-865a-3049a07ae5a6"> | <img width="1715" alt="Screenshot 2024-06-17 at 11 40 44 AM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/4b357e32-8e80-43a6-a6cc-213eaa75c5e3"> |

It's a minor nit for now. I realize it would require not relying on the `DashboardTable`'s empty state.

Reply:
Thanks, I also removed the search box as it serves no purpose with the limited view available.

![CleanShot 2024-06-17 at 09 14 47](https://github.com/ChildMindInstitute/mindlogger-admin/assets/2916156/74f8d98a-1166-45e2-92a8-d7259d1ad4c6)